lib/games_teams_collection.rb

```diff
@@ -28,4 +28,16 @@ def home_wins
   def percentage_home_wins
     ((home_wins / total_home_games.to_f) * 100).round(2)
   end
+
+  def max_goals
+    @games_teams.map {|game_team| game_team.goals.to_i}.max
+  end
+
+  def min_goals
+    @games_teams.map {|game_team| game_team.goals.to_i}.min
+  end
+
+  def biggest_blowout
+    max_goals - min_goals
+  end
```

Comment:
Comment for later, as we talked about in person: I think this -might- not be what the Iteration 2 instructions are calling for, might be looking for biggest difference in a given game.

Reply:
I concur with DF